Source/WebCore/page/Quirks.cpp

```diff
@@ -177,6 +177,22 @@ bool Quirks::hasBrokenEncryptedMediaAPISupportQuirk() const
     return m_hasBrokenEncryptedMediaAPISupportQuirk.value();
 }
 
+bool Quirks::shouldDisableContentChangeObserver() const
+{
+    if (!needsQuirks())
+        return false;
+    
+    auto& topDocument = m_document->topDocument();
+    
+    auto host = topDocument.url().host();
+    auto isYouTube = host.endsWith(".youtube.com"_s) || host == "youtube.com"_s;
+    
+    if (isYouTube && (topDocument.url().path().startsWithIgnoringASCIICase("/results"_s) || topDocument.url().path().startsWithIgnoringASCIICase("/watch"_s)))
```

Comment:
>topDocument.url().path().startsWithIgnoringASCIICase("/watch"
would this disable content observer on pages with playing video content? Do video controls (and cc, settings etc) work fine after this change?

Reply:
Yes, and yes, the controls work the same as without it.